tests/tamr_client/fake_json/categorization/test_project/test_manual_labels.json

```diff
@@ -5,6 +5,7 @@
       "path": "projects/2/unifiedDataset"
     },
     "response": {
+      "status": 200,
```

Comment:
For all of the `test_x.json` files, I filled in the status according to what is described on the swagger docs for Tamr, however, I am not confident these are correct in actual operation.  For instance, I think many that are listed as `200` are actually `204`.  This might not matter, as long as these tests are not relying on the response codes.